spec/controllers/api/event_hours_controller_spec.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+describe Api::EventHoursController, type: :controller do
+  let(:date) { Date.today.to_s }
+  let(:event_date) { create(:event_date, date: date.delete('-')) }
+  let!(:event_hour) { create(:event_hour, event_date: event_date) }
+
+  context 'with event dates by event_date_id' do
+    before do
+      get "api/event_dates/#{event_date.id}/event_hours"
+    end
+
+    it 'returns event hours of a specific event date' do
+      json_response = JSON.parse(response.body)
+      response = json_response['event_date']['event_hours']
+
+      expect(response.count).to eq(1)
```

Comment:
Not sure exactly what this code does. But if we are counting how many event_hours are coming back for an event date it may be 1, but most of the time (90%+?) it would be >1.